src/zerocash/Coin.cpp

```diff
@@ -73,7 +47,8 @@ Coin::Coin(const std::string bucket, Address& addr): addr_pk(), cm(), rho(ZC_RHO
     this->rho = rho_v;
     this->addr_pk = addr.getPublicAddress();
 
-    std::vector<unsigned char> a_pk = addr.getPublicAddress().getPublicAddressSecret();
```

Comment:
All of the above code is being removed. The new ordering of the spec is preserved in my WIP code. I'm avoiding changes to ordering and other such things in this PR.